modules/apps/static/portal-osgi-web/source-formatter-suppressions.xml

```diff
@@ -5,6 +5,7 @@
 		<suppress checks="MethodNamingCheck" files="portal-osgi-web-servlet-jsp-compiler/src/main/java/com/liferay/portal/osgi/web/servlet/jsp/compiler/internal/JspServlet\.java" />
 	</checkstyle>
 	<source-check>
+		<suppress checks="JavaPackagePathCheck" files="portal-osgi-web-portlet-container-test/src/testIntegration/java/com/liferay/portal/osgi/web/portlet/container/embedded/test/TestEmbeddedPortletDisplayTemplateHandler\.java" />
```

Comment:
Add an exclusion for test class for checking package ending with `.portlet.display.template`